docs/source/changelog.rst

```diff
@@ -7,6 +7,91 @@
 Changelog
 =========
 
+0.17.0
+------
+
+Breaking changes
+~~~~~~~~~~~~~~~~
+
```

Comment:
A default option to `xt::random::choice` changed so I think it should theoretically appear here

Reply:
Indeed, good catch!